faster_whisper/transcribe.py

```diff
@@ -79,7 +81,7 @@ def __init__(
         device: str = "auto",
         device_index: Union[int, List[int]] = 0,
         compute_type: str = "default",
-        cpu_threads: int = 0,
+        cpu_threads: int = 16,
```

Comment:
Maximum speed was achieved when using 16 in benchmarking. As you mentioned, if it is 0, it will decide on the number of cores based on OMP_NUM_THREADS. Changed as suggested.